rpm/0087-sailfishos-gecko-Fix-memory-reporting-of-wasm-memory.patch

```diff
@@ -0,0 +1,276 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Ryan Hunt <rhunt@eqrion.net>
+Date: Tue, 7 Sep 2021 21:03:48 +0000
+Subject: [PATCH] [sailfishos][gecko] Fix memory reporting of wasm memory.
+ Fixes JB#56908
+
+Backported from upstream gecko 94, bug 1615988.
```

Comment:
Please add link to the https://github.com/sailfishos-mirror/gecko-dev/commit/c871f743159fbbf2e1b22d877ace761360d72c03 as well.

Reply:
Please add this git mirror link to the git commit message as well (https://github.com/sailfishos/gecko-dev/pull/135/commits/971ade0fc05e78d16740565a9e91c3daa4ff6baf)

Reply:
Thanks, I've added it to both the commit summary and the patch description.